src/jit/simd.cpp

```diff
@@ -2354,13 +2354,20 @@ GenTree* Compiler::impSIMDIntrinsic(OPCODE                opcode,
 {
     assert(featureSIMD);
 
-    // Exit early if we are either not in one of the SIMD types or if the method
-    // is not a JIT Intrinsic (which requires the [Intrinsic] attribute).
-    if (!isSIMDClass(clsHnd) || ((methodFlags & CORINFO_FLG_JIT_INTRINSIC) == 0))
+    // Exit early if we are not in one of the SIMD types.
+    if (!isSIMDClass(clsHnd))
     {
         return nullptr;
     }
 
+#ifdef FEATURE_CORECLR
```

Comment:
This change was needed after #21601, since the desktop neither has `CORINFO_FLG_JIT_INTRINSIC`, nor does the `System.Numerics.Vectors.dll` on desktop have these marked.